src/components/views/rooms/RoomTile2.tsx

```diff
@@ -180,25 +182,25 @@ export default class RoomTile2 extends React.Component<IProps, IState> {
                             <ul>
                                 <li>
                                     <AccessibleButton onClick={(e) => this.onTagRoom(e, DefaultTagID.Favourite)}>
-                                        <span className="mx_IconizedContextMenu_icon mx_RoomTile2_iconStar" />
+                                        <span className="mx_IconizedContextMenu_icon mx_RoomTile2_iconStar"/>
```

Comment:
why did all the spaces before the closing of the tag get removed here? our eslint says not to do this :'(

Reply:
because no matter how much I try and convince WebStorm to not be annoying it wants to be annoying. I'm also highly not convinced the space will be the death of us.

Reply:
Sure, but it'll just cause more mess for once we sort the eslint/tslint rules :(